apps/dashboard/src/app/account/billing/checkout/checkout.component.ts

```diff
@@ -479,4 +453,19 @@ export class CheckoutComponent implements OnDestroy, OnInit {
       'noreferrer',
     );
   }
+
+  modifyBudget() {
+    this.billingService.saveBudget(this.budget?.value || null).subscribe({
+      next: ({ budgetBelowTotal }) => {
+        if (budgetBelowTotal) {
+          this.toaster.warning('billing.budget_warning');
```

Comment:
try to be consistent in the translation keys: in most cases we're using `-` instead of `_`. Don't hesitate to update the old keys you see, replacing `_` with `-` (or with a better tree structure when appropriate).

Reply:
ok, I'll update old keys if I find any